content/lessons/chapter-6/put-it-together-2.tsx

```diff
@@ -1,34 +1,164 @@
 'use client'
-
+import {ScriptingChallenge,Table,Introduction,Text,LessonInfo} from 'ui'
+import { useState } from 'react'
 import { useTranslations } from 'hooks'
-import { Introduction, Text } from 'ui'
+import { EditorConfig } from 'types'
+import { getLessonKey } from 'lib/progress'
 
 export const metadata = {
   title: 'chapter_six.put_it_together_one.title',
   image: '/assets/images/chapter-4-intro-1.jpg',
   key: 'CH6PUT2',
 }
 
+const javascript={program:` 
+   const tx = new Transaction();
+  
+ console.log(tx.serialize().toString('hex'));
+
+`,
+defaultFunction:{
+  name:"put-it-together-2",
+  args:["args"]
+},
+defaultCode:`class Transaction{
```

Comment:
before this line we will want to use this import statement `from struct import pack`